server/auth/check-jwt.js

```diff
@@ -18,3 +18,24 @@ const checkJwt = jwt({
 module.exports = {
   checkJwt
 }
+
```

Comment:
@Terieyenike , did you mean to add all of this commented code too?

Reply:
not really. I didn't work on the commented code.

Reply:
Can you remove the commented code from this pull request?

Reply:
Done.